go.mod

```diff
@@ -49,30 +50,5 @@ replace (
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20190923180330-3b6373338c9b
 	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
 	google.golang.org/grpc => google.golang.org/grpc v1.26.0 // https://github.com/etcd-io/etcd/issues/11563
-
-	// Pin to kube 1.16
```

Comment:
Could you add this comment back and keep the pinned requirements separate?

Reply:
@awgreene we're getting rid of pinned kube requirements. Please refer to [this](https://github.com/operator-framework/operator-lifecycle-manager/pull/1370#issuecomment-599712543) comment.

Reply:
I should have double checked the previous comment, my bad.